source/common/tcp_proxy/tcp_proxy.cc

```diff
@@ -239,6 +239,12 @@ void Filter::initialize(Network::ReadFilterCallbacks& callbacks, bool set_connec
   getStreamInfo().setDownstreamLocalAddress(read_callbacks_->connection().localAddress());
   getStreamInfo().setDownstreamRemoteAddress(read_callbacks_->connection().remoteAddress());
   getStreamInfo().setDownstreamSslConnection(read_callbacks_->connection().ssl());
+  read_callbacks_->connection().streamInfo().setDownstreamLocalAddress(
```

Comment:
Is this being upstreamed?

Reply:
I will after getting feedback from our team...

Reply:
@jplevyak : John removed tcp proxy change her and added to envoy only first, will merge here after that... https://github.com/envoyproxy/envoy/pull/9949

Can you please help merge this PR?